project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module

```diff
@@ -100,6 +119,12 @@ function c4m_features_og_group_dashboard_c4m_content_actions_info() {
       'callback'  => 'c4m_features_og_group_dashboard_action_promote',
       'weight'    => '2',
     ),
+
+    'action_recommend' => array(
+      'context'   => array('group', 'global'),
```

Comment:
No, I checked, when I delete the `group` the recommend flag doesn't show on the group nodes.